src/stories/select.stories.mdx

```diff
@@ -83,6 +98,29 @@ For user triggered actions
   </Story>
 </Canvas>
 
+<Canvas>
+  <Story
+    name='Photos'
```

Comment:
Change this name to something more descriptive of what you're showcasing, maybe like "With Button" or something

Reply:
yup